app/services/users.js

```diff
@@ -28,6 +29,25 @@ exports.createUser = async ({ name, lastName, email, password }) => {
   }
 };
 
+exports.makeAdminUser = async email => {
+  try {
```

Comment:
This logic could be simplified, since the try is not necessary, you could return directly, I would expect something like

exports.makeAdminUser = email =>
  UserModel.update( ).catch(error => {

  });